plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -240,23 +240,36 @@ class GeneralServiceFormSection extends Component {
   }
 
   getPlacementConstraintsFields(data = []) {
-    const errors = this.props.errors || [];
-    const showParameterLabel = data.some((constraint) => {
-      return ![GROUP_BY, UNIQUE].includes(constraint.operator);
+    const constraintsErrors = findNestedPropertyInObject(
+      this.props.errors,
+      'constraints'
+    );
+    const showValueLabel = data.some((constraint) => {
+      return hasThirdField[constraint.operator];
     });
     const showRequiredLabel = data.some((constraint) => {
-      return [LIKE, MAX_PER_OPERATOR].includes(constraint.operator);
+      return isRequired[constraint.operator];
     });
 
     return data.map((constraint, index) => {
       let fieldLabel = null;
       let operatorLabel = null;
-      let parameterLabel = null;
+      let valueLabel = null;
       let padDeleteButton = false;
-      const showParameterField = ![GROUP_BY, UNIQUE]
-        .includes(constraint.operator);
-      const paramterIsRequired = [LIKE, MAX_PER_OPERATOR]
-        .includes(constraint.operator);
+      const showValueField = hasThirdField[constraint.operator];
+      const valueIsRequired = isRequired[constraint.operator];
+      const fieldNameError = findNestedPropertyInObject(
+        constraintsErrors,
+        `${index}.fieldName`
+      );
+      const operatorError = findNestedPropertyInObject(
+        constraintsErrors,
+        `${index}.operator`
+      );
+      const valueError = findNestedPropertyInObject(
+        constraintsErrors,
+        `${index}.value`
+      );
 
       if (index === 0) {
         fieldLabel = this.getConstraintField(
```

Comment:
Let's please use internal functional components instead methods like `getConstraintField` similar to the pod review screen.